integration-tests/bridges/environments/polkadot-kusama/bridges_polkadot_kusama.sh

```diff
@@ -290,6 +291,20 @@ case "$1" in
           "$GLOBAL_CONSENSUS_KUSAMA_SOVEREIGN_ACCOUNT" \
           $AHP_KSM_ED \
           true
+      create_pool \
+          "ws://127.0.0.1:9910" \
+          "//Alice" \
+          "$(jq --null-input '{ "parents": 1, "interior": "Here" }')" \
+          "$(jq --null-input '{ "parents": 2, "interior": { "X1": [{ "GlobalConsensus": "Kusama" }] } }')"
+      # Create liquidity in the pool
+      add_liquidity \
+          "ws://127.0.0.1:9910" \
+          "//Alice" \
+          "$(jq --null-input '{ "parents": 1, "interior": "Here" }')" \
+          "$(jq --null-input '{ "parents": 2, "interior": { "X1": [{ "GlobalConsensus": "Kusama"}] } }')" \
+          10000000000 \
+          10000000000 \
```

Comment:
I suspect that at the moment this is called, 0 KSM have been issued on PAH, so there won't be any funds to add liquidity. I guess we need to bridge initial KSM before adding liq. Or maybe force asset balance to an initial value (but that will likely explode when we try to move them back)

Reply:
@brenzi yes, as we discussed yesterday, this is fixed by genesis preset